src/pages/lists.js

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import { DragDropContext as dragDropContext } from 'react-dnd';
+import HTML5Backend from 'react-dnd-html5-backend';
+import PropTypes from 'prop-types';
+import List from '../components/list/';
+import DragSort from '../components/list/dragSort';
+import Button from '../components/button/';
+import Icon from '../components/icon/';
+import Input from '../components/input/';
+
+class Item extends React.Component {
+  state = {
+    textInput: ''
+  };
+
+  handleChange = (e) => {
+    this.setState({ textInput: e.target.value });
+  };
+
+  render() {
+    if (this.props.isEditing) {
+      return (
+        <div className="intellyo-list-item list-item-editing">
+          <Input
+            defaultValue={ this.props.text }
+            onChange={ this.handleChange }
+          />
+          <Button
+            onClick={ () => this.props.onUpdate(this.state.textInput) }
+          >
+            Save
+          </Button>
+        </div>
+      );
+    }
+
+    return (
+      <div className="intellyo-list-item">
+        { this.props.text }
+        { this.props.onRemove &&
+          <Icon
+            onClick={ this.props.onRemove }
+            icon="ion-ios-trash"
+          />
+        }
+      </div>
+    );
+  }
+}
+
+Item.displayName = 'Item';
```

Comment:
Could you please declare these with the `static` keyword?

Reply:
The only case when you have follow this approach is when you declared a pure functional component.

```
const Foo = () => (<div>Hello</div>)
```

Reply:
✅